### DISCUSSION TITLE
Icons

### DISCUSSION THREAD

**Discussion by aquilonem** [shirokumacode/overwatch-omnic-rewards#29] [Show and tell]
Idk but if you want you can upgrade icons :)
[tokens](https://user-images.githubusercontent.com/108771119/182736373-adbc2a1a-2a84-442d-9aa9-e3d0d21f9ccd.png)

**Comment by shirokumacode** [comment]
Sorry for the late response. I kinda don't want to put a direct copy of the official icons on the app for "various" reasons. That's why they are different with a showcase of my artistic skill  :)
